hardware/raspberrypi/revision-codes/README.md

```diff
@@ -0,0 +1,94 @@
+# Raspberry Pi revision codes
+
+Each distinct Raspberry Pi model revision has a unique revision code. You can look up a Raspberry Pi's revision code by running:
+
+```bash
+cat /proc/cpuinfo
+```
+
+The last three lines show the hardware type, the revision code and the Pi's unique serial number, for example:
+
+```
+Hardware    : BCM2835
+Revision    : a02082
+Serial      : 00000000765fc593
+```
+
+## Old-style revision codes
+
+The first set of Raspberry Pi revisions were given sequential hex revision codes from `0002` to `0015`:
+
+Code | Model | Revision | RAM             | Manufacturer
+---- | ----- | -------- | --------------- | ------------
+0002 | B     | 1.0      | 256 MB          | ?
+0003 | B     | 1.0      | 256 MB          | ?
+0004 | B     | 2.0      | 256 MB          | Sony UK
+0005 | B     | 2.0      | 256 MB          | Qisda
+0006 | B     | 2.0      | 256 MB          | Egoman
+0007 | A     | 2.0      | 256 MB          | Egoman
+0008 | A     | 2.0      | 256 MB          | Sony UK
+0009 | A     | 2.0      | 256 MB          | Qisda
+000d | B     | 2.0      | 512 MB          | Egoman
+000e | B     | 2.0      | 512 MB          | Sony UK
+000f | B     | 2.0      | 512 MB          | Qisda
+0010 | B+    | 1.0      | 512 MB          | Sony
+0011 | CM1   | 1.0      | 512 MB          | Sony UK
+0012 | A+    | 1.1      | 256 MB          | Sony UK
+0013 | B+    | 1.2      | 512 MB          | ?
+0014 | CM1   | 1.0      | 512 MB          | Embest
+0015 | A+    | 1.1      | 256 MB / 512 MB | Embest
```

Comment:
What source are you using for 0014 and 0015?
https://github.com/AndrewFromMelbourne/raspberry_pi_revision has these listed as Sony.